external/fv3fit/tests/reservoir/test_reservoir_computing_model.py

```diff
@@ -9,6 +9,7 @@
     ReservoirComputingModel,
     Reservoir,
     ReservoirHyperparameters,
+    HybridReservoirComputingModel,
 )
```

Comment:
Optional: you could add a test similar to the reservoir input mask [test](https://github.com/ai2cm/fv3net/blob/3556fa956f434b6ee89549354af9404472214c33/external/fv3fit/tests/reservoir/test_reservoir.py#L141) that changing the hybrid data in a masked region doesn't affect the final output.